articles/user-guide/libraries/machine-learning/basic-classification.md

```diff
@@ -13,80 +13,90 @@ uid: microsoft.quantum.libraries.overview.machine-learning.basics
 
 # Basic classification: Classify data with the QDK
 
-In this Quickstart, you will learn how to run a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK. 
+In this Quickstart, you will learn how to run a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK.
```

Comment:
It might be a little strange to have two successive statements start with "In this Quickstart" and "In this guide". I would suggest picking of one of the two words and starting the second sentence with a different construction.

Reply:
I see your point, it's a bit repetitive. I will change it to something else.